frontend/app/src/types/frontend-settings.ts

```diff
@@ -80,10 +83,16 @@ const RoundingMode = z
 
 export type RoundingMode = z.infer<typeof RoundingMode>;
 
+// TODO: Add validation for max minutes allowed using Constraints.MAX_MINUTES_DELAY
```

Comment:
Is there anything stopping us for doing this at the moment?

Reply:
Based on our chat, this will happen if user previously already set higher number before this update released. This is error when zod try to parse user settings 
![image](https://user-images.githubusercontent.com/26648140/144023024-8da63867-a03e-418f-ab79-604f0782fcf1.png)

Reply:
In this case I think you could use [preprocess](https://github.com/colinhacks/zod#preprocess) to do this check with the constraints. This way you could just force it to max if it bigger than that. Since preprocess runs before the parsing/validation it should solve this.